Playmate/View Controllers/Session Details View/SessionDetailsViewController.m

```diff
@@ -191,10 +242,9 @@ - (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSe
 }
 
 - (nonnull __kindof UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cellForItemAtIndexPath:(NSIndexPath *)indexPath {
-    
     PlayerProfileCollectionCell *cell = [collectionView dequeueReusableCellWithReuseIdentifier:@"PlayerProfileCollectionCell" forIndexPath:indexPath];
     
-    cell.userProfile = self.sessionDetails.playersList[indexPath.row];
+    cell.userProfile = self.sessionDetails.playersList[self.sessionDetails.playersList.count- indexPath.row-1];
```

Comment:
why the change here? 

Any reason for the -1.
Maybe add a comment, so you will remember why.

Are we confident we will not have an index out of bounds? or a negative number?